redhat-developer/podman-desktop-redhat-account-ext
# fix: save auth state between pd sessions in secure storage

Fix #39.

How to test:
1. run podman desktopn `yarn watch --extension-folder path/to/sso/ext
2. Login to redhat sso from Settings/Authentication
3. exit podman-desktop
4. run podman desktopn `yarn watch --extension-folder path/to/sso/ext
5. open Settings/Authentication, see provider shows status logged in
